pallets/omnipool/src/lib.rs

```diff
@@ -1312,6 +1315,21 @@ pub mod pallet {
 				Ok(())
 			})
 		}
+		/// Update TVL cap
+		///
+		/// Parameters:
+		/// - `cap`: new tvl cap
+		///
+		/// Emits `TVLCapUpdated` event when successful.
+		///
+		#[pallet::weight(<T as Config>::WeightInfo::set_asset_weight_cap())]
+		#[transactional]
+		pub fn set_tvl_cap(origin: OriginFor<T>, cap: Balance) -> DispatchResult {
+			T::AddTokenOrigin::ensure_origin(origin)?;
```

Comment:
rename this origin, add new one or at least document this behaviour

Reply:
renamed to AuthorityOrigin.